src/aux-server/aux-web/aux-auth/site/AuthApp/AuthApp.vue

```diff
@@ -156,9 +156,16 @@
             <md-dialog-title>Add Studio</md-dialog-title>
 
             <md-dialog-content>
-                <md-field>
+                <md-field :class="studioNameFieldClass">
                     <label>Studio Name</label>
-                    <md-input v-model="studioName"></md-input>
+                    <md-input v-model="studioName" required></md-input>
+                    <span v-if="showEnterStudioNameError" class="md-error"
+                        >Studio name is required</span
+                    >
+                    <span v-if="showInvalidStudioNameError" class="md-error">{{
+                        errorMessage
+                    }}</span>
+                    <span v-if="showCreateStudioError" class="md-error">{{ errorMessage }}</span>
```

Comment:
@TroyceGowdy Instead of separate spans, you could then use the FieldErrors component for all of them at once.
Here's an example:
https://github.com/casual-simulation/casualos/blob/ea4bf2400b23d366772c554abc8fabee3942ce11/src/aux-server/aux-web/shared/vue-components/EnterAccountInfoDialog/EnterAccountInfoDialog.vue#L59-L69

You also have to add the FieldErrors component in the code behind:
https://github.com/casual-simulation/casualos/blob/e7ff3679133d7b92ef5bbcf6af0a6cc79fd75a8f/src/aux-server/aux-web/shared/vue-components/EnterAccountInfoDialog/EnterAccountInfoDialog.ts#L28-L35